tools/depends/configure.ac

```diff
@@ -301,6 +316,11 @@ case $host in
     platform_cxxflags="$platform_cflags"
     platform_os="linux"
     ffmpeg_options_default="--enable-vaapi --enable-vdpau --cpu=$use_cpu"
+    if [[[ "$host" =~ "i"."86" ]]]; then
+      use_cmake_cpu=${host:0:4}
```

Comment:
Have changed to extract the string from host, then can also e.g. i386 be given.

Reply:
There is no regex or other string manipulation needed. There exist one variable which always contains the right value.